### PR TITLE
Replaced prepend with insertBefore. Fixes #32

### LINKED ISSUE #32: Ellie does not work on Edge
SSCCE

1. Open https://ellie-app.com/new in Microsoft Edge browser.
2. Hit Compile
3. Compilation fails with message "Object doesn't support property or method 'prepend'"

ellie-edge

- [x] An [SSCCE](http://sscce.org/) that reproduces the problem
- [x] A screenshot of the problem in action

### PR DESCRIPTION
**Pull request by shoemakerdr**
Sorry for the wait on this PR. Let me know if you need me to revise.

Thanks!

**Please consider discussing complicated changes in [Slack](https://elmlang.slack.com/#ellie) [(join)](http://elmlang.herokuapp.com) or [via email](mailto:ellie-app@lukewestby.com) before submitting a PR**

### New Feature or UX change?

- Sign up for the Elm slack at http://elmlang.herokuapp.com and join the [#ellie channel](https://elmlang.slack.com/#ellie)
- Alternatively, send an email to ellie-app@lukewestby.com
- Have a conversation about your idea!
- New features and changes to Ellie's UX are considered thoroughly and carefully. An unannounced pull request is not the correct way to propose them! Ellie is managed in the same spirit as Elm itself. You can find out more in [this talk](https://www.youtube.com/watch?v=DSjbTC-hvqQ)

### Bug Fix?

- [x] Reference the [issue](https://github.com/lukewestby/ellie/issues) your PR fixes
- [ ] If your PR fixes an unreported issue, please open an issue first so that maintainers can have a chance to understand whether a PR is needed or if a deeper problem exists
- [ ] If your fix includes a lot of changed lines or files then a maintainer may ask for a short explanation of why the changes your made are the best way to fix the problem

**Thank you for contributing by fixing a problem with Ellie!**

### Performance Improvement?

 - [ ] Please include performance measurements if you can!

**Thank you for contributing by making Ellie faster!**

### CODE REVIEW

File: client/src/Pages/Editor/fixHtml.js [hunk 1]
```diff
@@ -91,7 +91,8 @@ export default ({ htmlCode, embedApi, sourceScript }) => {
 
       const div = doc.createElement('div')
       div.innerHTML = sourceScript
-      doc.body.prepend(div.children[0])
+      firstChild = doc.body.children[0]
+      doc.body.insertBefore(div.children[0], firstChild)
```
[the remark this answers is not in the document]
Comment (shoemakerdr) [reply client/src/Pages/Editor/fixHtml.js]: Looking at [docs on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore), it says: 

"If the reference node is null, the specified node is added to the end of the list of children of the specified parent node."

I tested this with both null/undefined, and it worked perfectly.

File: client/src/Pages/Editor/fixHtml.js [hunk 2]
```diff
@@ -91,7 +91,8 @@ export default ({ htmlCode, embedApi, sourceScript }) => {
 
       const div = doc.createElement('div')
       div.innerHTML = sourceScript
-      doc.body.prepend(div.children[0])
+      firstChild = doc.body.children[0]
```
[the remark this answers is not in the document]
Comment (shoemakerdr) [reply client/src/Pages/Editor/fixHtml.js]: Yikes, I don't know how I forgot this. I'll fix this now. Sorry!

[the remark this answers is not in the document]
Comment (shoemakerdr) [reply client/src/Pages/Editor/fixHtml.js]: Writing too much elm and python lately. Sorry about that.

File: client/src/Pages/Editor/fixHtml.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukewestby) [reply client/src/Pages/Editor/fixHtml.js]: cool!

File: client/src/Pages/Editor/fixHtml.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukewestby) [reply client/src/Pages/Editor/fixHtml.js]: me too! thank you!

**Pull request merged**